src/vue-components/views/manageGridsView.vue

```diff
@@ -285,13 +285,45 @@
                     this.resetFileInput(event);
                     return;
                 }
+
+                // Parse file to get import data for success message
+                let importData = await dataService.convertFileToImportData(importFile);
```

Comment:
you can change the method dataService.importBackupUploadedFile() below to not return a void promise, but one which returns the imported data. This way we won't have to calculate the "importData" two times, like now (bad performance)